linux/deploy_vm/deploy_vm.yml

```diff
@@ -36,6 +36,10 @@
     # Get SSH public key from localhost
     - include_tasks: ../utils/get_local_ssh_public_key.yml
 
+    - name: "Set fact of vm_exists to False"
```

Comment:
vm_exisits will be set in env_setup. Why add such code here?

Reply:
You are right. Will remove it.